transport/src/commonTest/kotlin/com/genesys/cloud/messenger/transport/utility/TestValues.kt

```diff
@@ -136,6 +136,11 @@ object QuickReplyTestValues {
     internal const val PAYLOAD_B = "payload_b"
     internal const val QUICK_REPLY = "QuickReply"
     internal const val BUTTON_RESPONSE = "ButtonResponse"
+    val postbackButtonResponse = ButtonResponse(
+        text = "Book Now",
```

Comment:
Would be nice to have text, type, payload as const vals somewhere, so that it will be easier to validate the values later on. see `buttonResponse_a`, also makes sense to put it under CardTestValues and make it internal.